packages/avatar/src/Avatar.stylesheet.js

```diff
@@ -8,14 +8,18 @@ const SizeMapping = {
   [sizes.XLARGE_64]: "extraLarge"
 };
 
+export function getBGColor(backgroundId, themeData) {
+  return themeData[`avatar.backgroundColor${backgroundId || 1}`];
+}
+
 export default function stylesheet(props, themeData) {
   const { backgroundId, size, stylesheet: customStylesheet } = props;
   const sizeString = size ? SizeMapping[size] : SizeMapping[sizes.MEDIUM_32];
   const diameter = themeData[`avatar.${sizeString}.diameter`];
   const fontSize = themeData[`avatar.${sizeString}.fontSize`];
   const fontFamily = themeData["basics.fontFamilies.main"];
-  const bgColor = themeData[`avatar.color${backgroundId || 1}.backgroundColor`];
-  const fgColor = themeData[`avatar.color${backgroundId || 1}.fontColor`];
+  const bgColor = getBGColor(backgroundId, themeData);
+  const fgColor = themeData["avatar.fontColor"];
```

Comment:
Hello there!

I don't believe this is a change we want. May I ask what is the intended goal here?

Thanks for contributing!

Reply:
Hi Stacey, do you mean the getBGColor function? If so, I agree and I've removed it in a folow-up commit.

Reply:
If you're referring to the overall color update including the font color change, these are based on the approved Figma board here: https://www.figma.com/file/S657OgrT8eiU4TreWWYQi8/LIB_Components?node-id=528%3A870

Reply:
I'm referring to the change to fgColor that no longer takes in a variable.

Reply:
I see! All of the new colors for the Avatar background were selected to be dark enough to not need the "light background" color, so there is now only one option for the font color, which used to be the "dark background' font color.

Reply:
Great, thanks for clarifying! :)